### PR TITLE
fix: view on popup

### PR DESCRIPTION
**Pull request by agungfir10**
This PR is change style on popup, in my problem is view on popup very disgust...

Problem

Solved:
Popup Redux is Right

### CODE REVIEW

**Review by Methuselah96 — CHANGES_REQUESTED** [review]
I believe doing this would break the responsive layout as demonstrated in https://github.com/reduxjs/redux-devtools/issues/1294, which was why the `min-width` was reverted back to `350px` in https://github.com/reduxjs/redux-devtools/pull/1365. Can you verify this doesn't break the responsive layout?

File: extension/src/style.pug [hunk 1]
```diff
@@ -7,7 +7,7 @@ style.
     overflow: hidden;
     height: 100%;
     width: 100%;
-    min-width: 350px;
+    min-width: 720px;
```
Comment (batya-bicha) [inline extension/src/style.pug:10]: maybe a 760px would be better, like it used to be?

Comment (Methuselah96) [reply extension/src/style.pug:10]: To be clear, the popup used to size to content which was almost always bigger than `min-width`. I suspect a change in Chrome 116 makes it so it no longer resizes to fit the content.

**Comment by Methuselah96** [comment]
Superseded by https://github.com/reduxjs/redux-devtools/pull/1468.

**Pull request closed without merging**